kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/IORecordWriter.java

```diff
@@ -0,0 +1,17 @@
+package io.confluent.connect.s3.storage;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import java.io.IOException;
+
+/**
+ * Interface which duplicates RecordWriter, yet with
+ * IOException throwing signatures.
+ */
+public interface IORecordWriter {
+  void write(SinkRecord var1) throws IOException;
```

Comment:
minor not: as this is an interface lets use a better variable name like `sinkRecord` or `record`